unverified/other/toxbot-git/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# Maintainer: Kevin MacMartin <prurigro at gmail dot com>
```

Comment:
Who's that guy? IIRC he was maintaining few other PKGBUILDs in AUR, right?

Since PKGBUILD will be maintained now in arch-repo-tox, he should be moved to ``Contributor``, and leave ``Maintainer`` empty or with ``?``, unless he'll explicitly confirm that he wants to continue maintaining PKGBUILD in this repo